recipes/Orange3-Textable/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "Orange3-Textable" %}
+{% set version = "3.1.11" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Orange3-Textable-{{ version }}.tar.gz
+  sha256: 87b070592d6f36bc31f6ae25812ce1b41d058b2a7bd974ef7c66d71319f8bf3a
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >3
+  run:
+    - appdirs
+    - chardet
+    - future
+    - lttl >=2.0.12
+    - orange3 >=3.14.0
+    - pycountry
+    - python >3
+    - setuptools
+    - treetaggerwrapper
+
+test:
+  imports:
+    - _textable
+    - _textable.widgets
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://textable.io
```

Comment:
Is this officially made by textable or thirdparty?

Reply:
This package is basically Textable's "product".